relay-server/src/actors/envelopes.rs

```diff
@@ -1755,7 +1755,7 @@ impl EnvelopeProcessor {
         mut state: ProcessEnvelopeState,
     ) -> Result<ProcessEnvelopeResponse, ProcessingError> {
         macro_rules! if_processing {
-            ($if_true:block $(, $if_not:block)?) => {
```

Comment:
random change: this is a little bit dangerous to allow in the macro, as we are throwing away the tokens that we accept here.